### PR TITLE
say3D Syntax Patch (#366)

### LINKED ISSUE #366: say3D - Update Syntax
Since Arma 3 Update 1.70 say3D allows the use of Array-Format, to get additional parameters for distance and pitch (see: https://dev.arma3.com/post/spotrep-00067). This should be reworked, in order to make sirens and other stuff audible at distances over 100m.

-> Fix already WIP, Pull Request will follow soon

----

Mission version: all

Game version: 1.70+

Branch: all

### PR DESCRIPTION
**Pull request by BlackFisch**
<Please review the guidelines for contributing to this repository. The link is above.>

Resolves #366 

#### Changes proposed in this pull request: 
- updated syntax of say3D Command to fit the Array-Syntax

### CODE REVIEW

**Comment by BoGuu** [comment]
Sorry for the late response. For the siren, have you found out the realistic distance it travels anyway? Is 500m too small?

**Comment by BlackFisch** [comment]
No, sorry... but I think 500m should be fine

File: Altis_Life.Altis/core/functions/fn_tazeSound.sqf [hunk 1]
```diff
@@ -8,4 +8,4 @@
 private ["_source"];
 _source = [_this,0,objNull,[objNull]] call BIS_fnc_param;
 if (isNull _source) exitWith {};
-_source say3D "tazerSound";
+_source say3D ["tazerSound",20,1];
```
Comment (Jawshy) [inline Altis_Life.Altis/core/functions/fn_tazeSound.sqf]: The tazerSound maxDistance is 20 metres, but the effective range is 35 metres for the stun pistol and 100 metres for the taser rifle. 

https://github.com/AsYetUntitled/Framework/blob/d8ad6bd7a067d92f23780ad0be637677085c3752/Altis_Life.Altis/core/functions/fn_handleDamage.sqf#L25-L28

**Comment by BlackFisch** [comment]
Thanks for the hint, will fix it :) give me one minute

**Comment by BlackFisch** [comment]
I hope this is fine for you? if I should change anything, just lt me know :)

**Comment by BlackFisch** [comment]
oops? Sorry for that and thanks for fixing my stupidness @Jawshy @Jason2605

**Comment by BlackFisch** [comment]
Should not be a problem :) just wait a sec

**Comment by DomT602** [comment]
@BlackFisch Looks good, only thing from me is to remove tazeSound from Config_SpyGlass (found here: https://github.com/AsYetUntitled/Framework/blob/master/Altis_Life.Altis/config/Config_SpyGlass.hpp#L267 )

**Comment by BlackFisch** [comment]
done @DomT602 :)

**Comment by DomT602** [comment]
@BlackFisch Thanks, @Jawshy this looks good to me now, sounds were increased in volume, I think they're fine, but don't know what they were like before, anyway code + cleanup is correct

**Comment by BlackFisch** [comment]
Problem was only, that the sounds were not audible over larger distances (like >50m) - that was the main reason for that pull request ;) @DomT602

**Comment by DomT602** [comment]
We just did some testing of the volumes, and we still can't hear harvesting, so if you could double the distance, it should be audible, and then we'll be good to merge.

**Comment by BlackFisch** [comment]
no problem, trying to help where ever I can ^^

**Pull request merged**